packages/console/src/containers/ConsoleContent/index.module.scss

```diff
@@ -14,4 +14,9 @@
   > * {
     @include _.main-content-width;
   }
+
+  // App Insights Wrapper on cloud env prevent the CSS assignment to direct children
+  #appInsightsWrapper > * {
```

Comment:
looks like `#appInsightsWrapper` is in the outer scope and it may have multiple wrappers at the same time?

curious why this will affect the inner code editor styles.